src/extensions/ConflictFixer/index.ts

```diff
@@ -47,9 +50,69 @@ export default class ConflictFixer implements Partial<Extension> {
         this.counter++;
 
         if (this.counter > 10) { // TODO: make configurable
+
             const msg = `[ShortNight]: Too many times(${this.counter}) of try fix conflict.`;
 
             if (DEBUG) {
+
+                /**
+                 * Holds the filter data from the event which have generated the conflicts
+                 * */
+                const filterData = ((ud) => {
+
+                    const eventsData = ud.map(ud => {
```

Comment:
Why don't we use `this.conflicts.event_body.map` as the first layer so that we don't need to filter `.filter(Boolean)`